implementation/jobberknoll/deno.json

```diff
@@ -1,13 +1,17 @@
 {
+  "version": "0.2.0",
```

Comment:
Consolidating separate versions for subpackages in the workspace to a single workspace version.